client/my-sites/people/people-profile/index.jsx

```diff
@@ -231,52 +242,58 @@ class PeopleProfile extends Component {
 		);
 	};
 
-	renderSubscribedDate = () => {
-		if ( ! this.props.user || ! this.props.user.date_subscribed ) {
+	const renderSubscribedDate = () => {
+		if ( ! user || ! user.date_subscribed ) {
 			return;
 		}
 
 		return (
 			<div className="people-profile__subscribed">
-				{ this.props.translate( 'Since %(formattedDate)s', {
+				{ translate( 'Since %(formattedDate)s', {
 					context: 'How long a user has been subscribed to a blog. Example: "Since Sep 16, 2015"',
 					args: {
-						formattedDate: this.props.moment( this.props.user.date_subscribed ).format( 'll' ),
+						formattedDate: moment( user.date_subscribed ).format( 'll' ),
 					},
 				} ) }
 			</div>
 		);
 	};
 
-	isFollowerType = () => {
-		return this.props.user && ! this.props.user.roles && this.props.user.date_subscribed;
+	const isFollowerType = () => {
+		return user && ! user.roles && user.date_subscribed;
 	};
 
-	render() {
-		const { user } = this.props;
+	const classes = classNames( 'people-profile', {
+		'is-placeholder': ! user,
+	} );
 
-		const classes = classNames( 'people-profile', {
-			'is-placeholder': ! user,
-		} );
-
-		return (
-			<div className={ classes }>
-				<div className="people-profile__gravatar">
-					<Gravatar user={ user } size={ 72 } />
-				</div>
-				<div className="people-profile__detail">
-					{ this.renderNameOrEmail() }
-					{ this.renderLogin() }
-					{ this.isFollowerType() ? this.renderSubscribedDate() : this.renderRole() }
-				</div>
+	return (
+		<div className={ classes }>
+			<div className="people-profile__gravatar">
+				<Gravatar user={ user } size={ 72 } />
 			</div>
-		);
-	}
-}
-
-export default connect( ( _state, { siteId, user } ) => {
-	const externalContributors = ( siteId && requestExternalContributors( siteId ).data ) || [];
+			<div className="people-profile__detail">
+				{ renderNameOrEmail() }
+				{ renderLogin() }
+				{ isFollowerType() ? renderSubscribedDate() : renderRole() }
+			</div>
+		</div>
+	);
+};
+
+PeopleProfile.propType = {
+	siteId: PropTypes.number.isRequired,
+	translate: PropTypes.func.isRequired,
+	moment: PropTypes.func.isRequired,
+	user: PropTypes.object,
+	type: PropTypes.string,
+	invite: PropTypes.object,
+};
+
+export default connect( ( state, ownProps ) => {
+	const siteId = getSelectedSiteId( state );
```

Comment:
The `siteId` prop used to be passed from the parent component, and the component didn't need to retrieve it from Redux itself. As the usages of `PeopleProfile` haven't changed, `siteId` continues to be passed even now. So why add `getSelectedSiteId` now? The component doesn't need it -- it doesn't need to be connected to Redux at all after this PR.